README.md

```diff
@@ -41,5 +46,6 @@ jobs:
       uses: okteto/create-namespace@master
       with:
         namespace: devenvs-cindylopez
+        share: cindylopez,rberrelleza,pchico83
```

Comment:
what happens if the owner of the namespace is on the list? and it isn't on the list?